tests/libtransmission/test-fixtures.h

```diff
@@ -396,8 +396,17 @@ class SessionTest : public SandboxedTest
     {
         auto verified_lock = std::unique_lock(verified_mutex_);
         auto const n_previously_verified = std::size(verified_);
-        auto* const tor = tr_torrentNew(ctor, nullptr);
 
+        tr_ctorSetVerifyDoneCallback(
+            ctor,
+            [this](tr_torrent* const tor)
+            {
+                auto verified_lock = std::scoped_lock(verified_mutex_);
```

Comment:
```suggestion
                auto verified_lock = std::lock_guard{ verified_mutex_ };
```

Reply:
fixed in 63e2fcaa304b63b5ed56dde5053813e236e22848